spec/rubocop/cop/performance/array_semi_infinite_range_slice_spec.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Performance::ArraySemiInfiniteRangeSlice, :config do
+  subject(:cop) { described_class.new(config) }
+
+  # TODO: The following is no longer required when RuboCop 0.78 or lower support will be dropped.
+  # https://github.com/rubocop-hq/rubocop/pull/7605
+  let(:ruby_version) { 2.7 }
```

Comment:
FYI, latest RuboCop Performance requires 0.87 or higher, so it can now be removed. Nice informative comment It is!